### ISSUE TITLE
skip pre-install check var needed for Clusters with multiple versions of MAS/Add-ons and not in a Ready State

### ISSUE THREAD

**Issue by andreawills** [ibm-mas/installer#759]
skip pre-install check variable is needed for Clusters with multiple versions of MAS/Add-ons where the pods are not always in a ready state.. Or where developers are working on projects where they pull their own versions .  

I am not sure if it should be a all or nothing type of pre-install check or only ignore the Projects labeled mas-*-core, mas-*-manage, etc..  but its needed for others can share one ocp cluster with various versions

Checking Deployments are healthy (122/200 retries with a 300 second delay)
[NOTREADY] mas-briang-manage/briang-masdev-all = 1 replicas/None ready/1 updated/None available

**Comment by rene-oromtz** [comment]
+1

Storage Fusion could also be benefited from this enhancement. As with previous discussion with Maximo team, current limitation for Storage Fusion B/R is that it requires AMQ Streams as 2.4.0.x version (this is being explored for future support to latest AMQ versions).
This means that Kafka cluster needs to be manually configured for Maximo applications that use this (IOT, Monitor, Manage). 
During mas install, KafkaCfg can be provided in masconfig dir so there is not much of an issue with that, however, during the pre-install-check, the process is getting stalled at Check Subscription Status

```
TASK [ibm.mas_devops.ocp_verify : Check Subscription Status] *******************
Checking Subscriptions are up to date (60 retries with a 300 second delay)
...
ibm-backup-restore/amq-streams-stable-redhat-operators-openshift-marketplace = AtLatestKnown
...
kafka/amq-streams = UpgradePending
...
Delaying 300 seconds before next check
```
![image](https://github.com/ibm-mas/cli/assets/157750458/ecddb447-e8ba-4a44-b142-5390dd2e16bf)

If customer creates AMQ Streams operator following [Maximo docs](https://www.ibm.com/docs/en/mas-cd/continuous-delivery?topic=dependencies-installing-apache-kafka) but setting the Update approval as manual and with version 2.4.0.0 to match the version for Fusion B/R, the pre check will detect that there is an update available for the operator and it will be looping until that operator is upgraded. 

Please note that in ibm-backup-restore namespace the operator is being shown as AtLatestKnown but due to how Fusion is built and to prevent in any case to upgrade this operator. However, if we create Kafka cluster in different namespace, this upgrade blocker will not be available and an install plan will be created and waiting for Manual approval. 

As current workaround, I can create new Kakfa cluster in ibm-backup-restore to skip this verification but this might not be suitable for all users. 

Also, the issue I'm describing is the same issue as in #727 so a skip pre-install-check might be of use

**Comment by andrercm** [comment]
Should be fixed in this PR: https://github.com/ibm-mas/cli/pull/893

Fix available in beta version `cli:8.1.0-pre.skip-pre-install`

**Comment by durera** [comment]
Delivered in https://github.com/ibm-mas/cli/pull/893
